src/utils/formatter.ts

```diff
@@ -76,3 +76,31 @@ export const formatOrgLogs = (allTasks: any[]) => {
 
   return obj
 }
+
+export const formatTxtLogs = (allTasks: string[]) => {
+  let obj: Record<string, formattedTaskType[]> = {}
```

Comment:
can this be named to record instead? and the function to know that it returns record as type of Object?